test/jiffy_relative_locale_test.dart

```diff
@@ -322,5 +322,22 @@ void main() {
       expect(jiffy12.fromNow(), 'bir yıl önce');
       expect(jiffy13.fromNow(), '10 yıl önce');
     });
+    test(
+        'test Jiffy.locale() method with parsing en locale should return correct date time in en locale',
```

Comment:
Please change the  test description from `should return correct date time in **en** locale` to `should return correct date time in **sv** locale`